addons/transifex/models/ir_translation.py

```diff
@@ -8,17 +8,58 @@
     from ConfigParser import ConfigParser
 from os.path import join as opj
 import os
+import re
 import werkzeug
 
-from odoo import models, fields
+from odoo import models, fields, _
 from odoo.modules.module import ad_paths
 
+REGEX_PLACEHOLDERS = re.compile("%(\(\w+\)|)([0+-]|)(\d+|\*|)[diouxXcrsa]|%(\(\w+\)|)([0+-]|)(\d+|\*|)(\.(\d+|\*|)|)[eEfFgG]|%%")
```

Comment:
Use rawstrings. Also probably worth using triple-quoted strings, re.VERBOSE and comments for this one as it's a bit gnarly.